thrust/thrust/system/cuda/execution_policy.h

```diff
@@ -37,3 +37,67 @@
 #endif // no system header
 #include <thrust/system/cuda/detail/execution_policy.h>
 #include <thrust/system/cuda/detail/par.h>
+
+// define these entities here for the purpose of Doxygenating them
+// they are actually defined elsewhere
+#if 0
+THRUST_NAMESPACE_BEGIN
+namespace cuda
+{
+
+/*! \addtogroup execution_policies
+ *  \{
+ */
+
+/*! \p thrust::cuda::par is the parallel execution policy associated with
+ *  Thrust's CUDA device backend.
+ *
+ *  Instead of relying on implicit algorithm dispatch through iterator system tags,
+ *  users may directly target Thrust's CUDA backend by providing \p thrust::cuda::par
+ *  as an algorithm parameter. The policy can be attached to a specific CUDA stream
+ *  using the `.on(stream)` helper.
+ *
+ *  The type of \p thrust::cuda::par is implementation-defined.
+ */
+static const unspecified par;
+
+/*! \p thrust::cuda::par_nosync is a parallel execution policy targeting
+ *  Thrust's CUDA device backend that allows algorithms to elide optional
+ *  stream synchronizations.
+ *
+ *  Similar to \p thrust::cuda::par, it allows execution of Thrust algorithms in
+ *  a specific CUDA stream via `.on(stream)`. In addition, \p thrust::cuda::par_nosync
+ *  indicates that an algorithm is free to avoid any synchronization of the associated
+ *  stream that is not strictly required for correctness. Algorithms may also return
+ *  before the corresponding kernels are completed (similar to asynchronous kernel launches).
+ *  The user must perform explicit synchronization when necessary.
+ *
+ *  Example:
+ *  \code
```

Comment:
It may be a good idea to start using `literalinclude`s for the Thrust example snippets too. That's a good starting point. See (2) https://github.com/NVIDIA/cccl/issues/2908